### PR TITLE
Era Timing

### PR DESCRIPTION
**Pull request by karthikeyan-cere**

### CODE REVIEW

File: scripts/tokenomics-emulation/network.ts [hunk 1]
```diff
@@ -143,6 +143,23 @@ class Network {
     });
   }
 
+  /**
+   * Calculate the remaining ERA time. 
+   * @returns Era Start time
+   */
+  public async eraTime() {
```
Comment (AndreiNavoichyk) [inline scripts/tokenomics-emulation/network.ts:150]: @karthikeyan-cere 
Rename the method and comments.

File: scripts/tokenomics-emulation/network.ts [hunk 2]
```diff
@@ -143,6 +143,23 @@ class Network {
...
+    console.log(`Calculating remaining ERA time`);
+    let era = await this.api.query.staking.currentEra();
```
Comment (AndreiNavoichyk) [inline scripts/tokenomics-emulation/network.ts:152]: Move it to the separate private method.

File: scripts/tokenomics-emulation/network.ts [hunk 3]
```diff
@@ -143,6 +143,23 @@ class Network {
     });
   }
 
+  /**
+   * Calculate the remaining ERA time. 
+   * @returns Era Start time
+   */
+  public async eraTime() {
+    console.log(`Calculating remaining ERA time`);
```
Comment (AndreiNavoichyk) [inline scripts/tokenomics-emulation/network.ts:151]: It's not calculating

File: scripts/tokenomics-emulation/network.ts [hunk 4]
```diff
@@ -143,6 +143,32 @@ class Network {
     });
   }
 
+  /**
+   *  Era Wait Time
+   * @returns Boolean after new era started
+   */
+  public async eraWaitTime() {
```
Comment (AndreiNavoichyk) [inline scripts/tokenomics-emulation/network.ts:150]: rename to `waitForANewEra`

Comment (karthikeyan-cere) [reply scripts/tokenomics-emulation/network.ts]: Renamed

File: scripts/tokenomics-emulation/network.ts (hunk 3 above)
Comment (karthikeyan-cere) [reply scripts/tokenomics-emulation/network.ts]: Removed

File: scripts/tokenomics-emulation/network.ts (hunk 2 above)
Comment (karthikeyan-cere) [reply scripts/tokenomics-emulation/network.ts]: Added a new method

File: scripts/tokenomics-emulation/network.ts (hunk 1 above)
Comment (karthikeyan-cere) [reply scripts/tokenomics-emulation/network.ts]: Renamed

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**